packages/integration-karma/helpers/test-utils.js

```diff
@@ -330,11 +330,25 @@ window.TestUtils = (function(lwc, jasmine, beforeAll) {
         register = {};
     }
 
+    // #986 - childNodes on the host element returns a fake shadow comment node on IE11 for debugging purposed. This method
+    // filters this node.
+    function getHostChildNodes(host) {
+        var filtered = [];
+        const childNodes = host.childNodes;
+        for (var i = 0; i < childNodes.length; i++) {
+            var n = childNodes[i];
+            if (!(n.nodeType === Node.COMMENT_NODE && n.tagName.startsWith('#shadow-root'))) {
+                filtered.push(n);
+            }
+        }
+        return filtered;
```

Comment:
Just saw that this used to be: ```
function getHostChildNodes(host) {
    return [...host.childNodes].filter(n => {
        return n.nodeType !== Node.COMMENT_NODE && !n.tagName.startsWith('#shadow-root');
    });
}
```
This was incorrect because we only wanted to filter out `#shadow-root` comment nodes, right?

Reply:
test-utils is not transpiled down to ES5, it is appended as-is. So will use something like this

 ```
return Array.prototype.slice.call(host.childNodes).filter(function(n) {
return !( n.nodeType === Node.COMMENT_NODE && n.tagName.startsWith('#shadow-root'))
});
```

Reply:
> This was incorrect because we only wanted to filter out #shadow-root comment nodes, right?
yes. It was failing if `n` was a text node.